### PR TITLE
Add tests for #532

### PR DESCRIPTION
**Pull request by vincentchalamon**
Add unit tests for #532

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #531
| License       | MIT

### CODE REVIEW

File: tests/Bridge/Symfony/Routing/IriConverterTest.php [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+            $propertyMetadataFactoryMock->reveal(),
+            $itemDataProviderMock->reveal(),
+            $routerMock->reveal()
+        );
+        $converter->getItemFromIri('/users/3');
+    }
+
+    /**
+     * @expectedException InvalidArgumentException
+     * @expectedExceptionMessage No resource associated to "".
```
Comment (soyuka) [inline tests/Bridge/Symfony/Routing/IriConverterTest.php]: `Failed asserting that exception message 'No resource associated to "/users/3".' contains 'No resource associated to "".'.`

I think url is missing

Comment (vincentchalamon) [inline tests/Bridge/Symfony/Routing/IriConverterTest.php]: Yep, I just fixed it 😉 
But another unit test is failing, out of my modifications (I guess this unit test should also fail on master)

Comment (soyuka) [inline tests/Bridge/Symfony/Routing/IriConverterTest.php]: About the warning? #542

Comment (Simperfit) [inline tests/Bridge/Symfony/Routing/IriConverterTest.php]: You have to clear the test symfony app's cache in order to have that test working ;)

**Comment by soyuka** [comment]
:+1:

File: tests/Bridge/Symfony/Routing/IriConverterTest.php (hunk 1 above)
Comment (vincentchalamon) [inline tests/Bridge/Symfony/Routing/IriConverterTest.php]: No it was about a unit test that was failing on another class, but I don't know why now it's ok :)

**Comment by dunglas** [comment]
Thank you @vincentchalamon

**Pull request merged**